src/tuples-and-arrays/match.md

```diff
@@ -7,7 +7,7 @@ minutes: 10
 The `match` keyword lets you match a value against one or more _patterns_. The
 comparisons are done from top to bottom and the first match wins.
 
-The patterns can be simple values, similarly to `switch` in C and C++:
```

Comment:
I think this whitespace was correct as it was

Reply:
ok good.